README.md

```diff
@@ -27,27 +47,36 @@ You will need the following tools to clone, build, and run Glimesh:
 - **nodejs / npm**: Front-end package management
 - **inotify-tools**: Filesystem monitoring dependencies for developer convenience (watching changes)
 
-On modern versions of Ubuntu, you can install these packages with the following command:
-
-```sh
-sudo apt install git esl-erlang elixir postgresql npm inotify-tools
-```
+You may need to translate these exact dependencies into their appropriate names for your OS distribution.
 
-Other distributions likely have packages available for these tools as well.
-
-## Cloning
+### Cloning
 
 To clone a local copy of Glimesh.tv, run
 
 ```sh
-git clone git@github.com:Glimesh/glimesh.tv.git
+git clone https://github.com/Glimesh/glimesh.tv.git
+# or if you have permissions to the repository
```

Comment:
i would say change this to say `or if you want to use ssh key auth to git for the repo` as the ssh path is only if you are logged in to git on a global level and not repo level

Reply:
Updated as well